src/test/javascript/spec/component/course/course-group.component.spec.ts

```diff
@@ -28,6 +28,15 @@ import { MockRouterLinkDirective } from '../../helpers/mocks/directive/mock-rout
 import { UsersImportButtonComponent } from 'app/shared/import/users-import-button.component';
 import { TranslateDirective } from 'app/shared/language/translate.directive';
 import { AlertService } from 'app/core/util/alert.service';
+import { EMAIL_KEY, NAME_KEY, REGISTRATION_NUMBER_KEY, USERNAME_KEY } from 'app/course/course-scores/course-scores.component';
+
+const generateCsv = jest.fn();
+
+jest.mock('export-to-csv', () => ({
+    ExportToCsv: jest.fn().mockImplementation(() => ({
+        generateCsv,
+    })),
+}));
```

Comment:
```suggestion
jest.mock('export-to-csv', () => ({
    ExportToCsv: jest.fn().mockImplementation(() => ({
        generateCsv: jest.fn(),
    })),
}));
```
The variable is never used, so just assigning the value inside the object is enough.

Reply:
Thanks for the comment. I have followed your sugestion.